rt-forecast/update-format-rt.R

```diff
@@ -68,11 +77,29 @@ region_forecasts <- region_forecasts %>%
 
 region_forecasts$forecast_date <- forecast_date
 
-# Save forecast -----------------------------------------------------------
+# Save submission
 # Dated
 readr::write_csv(region_forecasts,
                  paste0("rt-forecast/submission-files/dated/", forecast_date, "-rt-forecast-submission.csv"))
-#Latest
+# Latest
 readr::write_csv(region_forecasts,
                  paste0("rt-forecast/submission-files/latest-rt-forecast-submission.csv"))
 
+
+# Get and save samples ------------------------------------------------------------
+
+region_forecasts_samples <- purrr::map2_dfr(.x = forecasts, .y = names(forecasts),
+                                    ~ format_rt_forecast(loc = .x, loc_name = .y,
+                                                         forecast_date = forecast_date,
+                                                         forecast_adjustment = 11 + 5,
+                                                         horizon_weeks = 5,
+                                                         state_data_cumulative = state_data_cumulative,
+                                                         state_data_daily = state_data_daily, 
+                                                         samples = TRUE,
+                                                         replace_missing_with_latest = FALSE
```

Comment:
I've set this replace_missing_with_latest argument as FALSE. I'm not actually sure why you would want to revert to the latest forecast if an older forecast is missing - I think I don't understand this?! :) No problem if you do need to of course - just change it to TRUE.

Reply:
the only reason would be so that you can just run it on any date and just get the latest and I wanted to have a fallback option. But yeah, fine to have it the way you changed it :)